Nos/Views/RepostButton.swift

```diff
@@ -62,6 +63,9 @@ struct RepostButton: View {
             Button(String(localized: .localizable.repost)) {
                 Task { await repostNote() }
             }
+            Button(String(localized: .localizable.quote)) {
+                quoteNote()
```

Comment:
I thin we can just do `showQuotedNoteComposer = true` here given that it is not reused from another place